src/brackets.js

```diff
@@ -107,6 +107,7 @@ define(function (require, exports, module) {
     require("utils/Resizer");
     require("LiveDevelopment/main");
     require("utils/ColorUtils");
+    require("utils/BorderRadiusUtils");
```

Comment:
We shouldn't need to `require()` a file needed by an extension in the core.  Can you not move this into your `main.js` or the extension?

Reply:
@humphd  Should I remove this line from the brackets.js file?

Reply:
Yeah, I think so.  The only reason to have it here is if you have to have it loaded early for some reason so that other code can access it, and it has to do some kind of initialization.  In your case, you should just load it when you need it, where you need it.

Reply:
In the brackets.js, the colorUtil.js file is also being requried(), That's why I put the borderRadiusUtil.js in here. And I noticed that, if I remove the borderRadiusUtil.js file away from bracket.js, then the BorderRadiusUtil.js will not be loaded in the browser. So basically, if I remove the file from brackets.js, then in my extension, for instance, the InlineBorderRadius.js will not get the BorderRadiusUtils.js because it is using something like this  " brackets.getModule("utils/BorderRadiusUtils"); " to load the util files.

Reply:
Both `require()` and `brackets.getModule()` do the same thing, namely, they load a module.  However, you use one or the other depending on where the module is defined.  For example, in the core of Brackets, you always use `require()`.  In an extension, you use `require()` when the module you are loading is defined within the extension.  However, if you need to load a core module in an extension, you use `brackets.getModule()`.

If you only need this file in your extension code, which you do, you can just use `require()` in the extension code before it is used.

Reply:
I got it thank you, so I think instead of using brackets.getModule, I should use require() to do the samething in my extension files, because I'm only getting reference to that one file , it's not nessasary to put the file in the core module. is this correct?

Reply:
You got it.  Let me know if you need help getting it to load properly--sometimes there can be dependency load-order issues with require.